src/providers/krb5/krb5_child.c

```diff
@@ -3154,6 +3158,49 @@ static int k5c_setup(struct krb5_req *kr, uint32_t offline)
     return kerr;
 }
 
+static krb5_error_code validate_keytab(struct krb5_req *kr)
```

Comment:
Hi,

`validate_keytab` might be a bit misleading, it might be better to use `validate_keytab_name`, `ensure_keytab_name`, `check_keytab_name` or similar to indicate that it is only about the name not about keytab content.

bye,
Sumit

Reply:
Agree